src/components/charts/ChartWrapper.tsx

```diff
@@ -0,0 +1,121 @@
+import React from "react"
+import Chart from "react-apexcharts"
+import merge from "deepmerge"
+import { globalOptions } from "../shared/ApexConfig"
+
+import { BarSeries } from "./BarChart"
+import { LineSeries } from "./LineChart"
+
+type Series = BarSeries | LineSeries
+
+export type RecordEntry = Record<string, string | number>
```

Comment:
As a matter of practice, does TypeScript put types into those `.d` files instead of in the `.tsx` files?

Reply:
I think so. That's also something that I need to figure out. At first I wasn't sure if Storybook can produce documentation with type declarations in other files, but turns out it does a pretty good job